README.md

```diff
@@ -37,45 +38,44 @@ This necessitates additional steps in the protocol to handle requesting further
+A desirable feature of UCAN is that it is interpretable by systems that understand existing token formats based on JWT. For instance, most backend web frameworks come with an authorization plugin. These typically depend on auth tokens being provided as [`Authorization: Bearer`][RFC 6750].
 
 # 2 Request Headers
 
-The HTTP headers MUST include an `Authorization: Bearer` header, and MAY include zero or more `ucan` fields that form a CID-to-UCAN table.
+The HTTP headers MUST include an `Authorization: Bearer` header, and MAY include zero or one `ucans` field that contains a UCAN array.
 
-| Header                  | Type             | Description                  | Required |
-| ----------------------- | ---------------- | ---------------------------- | -------- |
-| `Authorization: Bearer` | `ucan-jwt`       | Entry point "top-level" UCAN | Yes      |
-| `ucan`                  | `[cid:ucan-jwt]` | Mapping of CIDs to UCANs     | No       |
+| Header                  | Type         | Description                  | Required |
+| ----------------------- | ------------ | ---------------------------- | -------- |
+| `Authorization: Bearer` | `ucan-jwt`   | Entry point "top-level" UCAN | Yes      |
+| `ucans`                 | `[ucan-jwt]` | UCAN proof array             | No       |
 
 ## 2.1 Entry Point
 
-The entry-point for a UCAN chain MUST be `Authorization: Bearer <ucan>`. This is the only REQUIRED field. The UCAN contained in this field MUST be [encoded as a JWT](https://www.rfc-editor.org/rfc/rfc7519#section-3). Per the UCAN spec, this token SHOULD be unique per request.
+The entry-point for a UCAN chain MUST be `Authorization: Bearer <ucan>`. This is the only REQUIRED field. The UCAN contained in this field MUST be encoded as a [JWT]. Per the UCAN spec, this token SHOULD be unique per request.
 
 ``` abnf
 ucan-entry-point = "Authorization:" 1*SP "Bearer" 1*SP <ucan-jwt>
 ```
 
-## 2.2 UCAN-by-CID Table
+## 2.2 UCAN Proof Array
 
-The entry point UCAN MAY contain CID references to further UCANs as "proofs" (values in the `prf` field). For the entry point UCAN to be valid, these MUST also be valid and available. Since UCANs MAY be cached in previous requests, including UCAN mappings in this table is OPTIONAL. Only one `ucan` header SHOULD be used. If more than one `ucan` header is present, merging the elements from all headers on read is RECOMMENDED.
+The entry point UCAN MAY contain CID references to further UCANs as "proofs" (values in the `prf` field). For the entry point UCAN to be valid, these MUST also be valid and available. Since UCANs MAY be cached in previous requests, including UCAN mappings in this table is OPTIONAL. Only one `ucan` header SHOULD be used. More than one `ucan` header MUST NOT be used.
```

Comment:
```suggestion
The entry point UCAN MAY contain CID references to further UCANs as "proofs" (values in the `prf` field). For the entry point UCAN to be valid, these MUST also be valid and available. Since UCANs MAY be cached in previous requests, including UCAN mappings in this table is OPTIONAL. Only one `ucans` header SHOULD be used. More than one `ucans` header MUST NOT be used.
```